js/components/Note/Note.js

```diff
@@ -5,6 +5,17 @@ import styles from './styles';
 import PropTypes from 'prop-types';
 import moment from 'moment';
 import UserContext from '../../context/UserContext';
+import gql from 'graphql-tag';
+import { Mutation } from 'react-apollo';
+
+const STAR_POST = gql`
```

Comment:
this structure should be followed throughout your project. Not just in your pull requests.